docs/src/summary.md

```diff
@@ -0,0 +1,34 @@
+# Intro
+
+factory_bot is a fixtures replacement with a straightforward definition syntax,
+support for multiple build strategies (saved instances, unsaved instances,
+attribute hashes, and stubbed objects), and support for multiple factories for
+the same class (user, admin_user, and so on), including factory inheritance.
+
+Its documentation is split as such:
+
+- the [guide](setup/summary.html) is a great place to start for first-time users.
+- the [cookbook](cookbook/has_many-associations.html) is the go-to place for creative solutions to common situations.
+- the [wiki](https://github.com/thoughtbot/factory_bot/wiki) details considerations for integrating with other software.
+- the [reference](ref/build-strategies.html) is terse facts for those who use this project often.
+
+## License
+
+factory_bot is Copyright © 2008-2022 Joe Ferris and thoughtbot. It is free
```

Comment:
Do we need to keep the final year here? Otherwise, we would need to update things every year 🤔 Not sure how these things work, though.